### ISSUE TITLE
[feature request] Expand window across multiple screens

### ISSUE THREAD

**Issue by JasMich** [swaywm/sway#5061]
I'm using a multi head setup with about 10 screens in portrait orientation. It would be great if it was possible to expand a single window over several of these screens.
Unfortunately `$mod + r` doesn't allow me to expand a window more than the current screen's size.

**Comment by Emantor** [comment]
```
fullscreen [enable|disable|toggle] [global]
     Makes focused view fullscreen, non-fullscreen, or the opposite of what it is now. If no argument is given, it does the same as toggle. If global is specified, the view will be fullscreen across all outputs.
```
from `man 5 sway`.

**Comment by JasMich** [comment]
Sorry, I incorrectly described: I want to use 6 of my 10 outputs for *full-screen*. Is that somehow possible?

**Comment by tokyovigilante** [comment]
Possibly a duplicate/case of #1666 (which I would also really like to see implemented).

**Comment by JasMich** [comment]
Yes, somehow. But a kind of some-monitor-full-screen was not mentioned.

Does someone know why normal global fullscreen is not working on firefox?

**Comment by tokyovigilante** [comment]
If you can merge outputs, then it will behave like a single output and fullscreen will (should?) automatically just span that virtual output.

**Comment by JasMich** [comment]
Where is that documented? That's kind of what I'm looking for...

**Comment by tokyovigilante** [comment]
That's what I'm getting at sorry, you can't currently merge outputs into a single virtual screen. That's what #1666 is about, but it's not currently implemented.

**Comment by 532910** [comment]
> Yes, somehow. But a kind of some-monitor-full-screen was not mentioned.
> 
> Does someone know why normal global fullscreen is not working on firefox?

https://blog.wains.be/2019/2019-11-13-full-screen-firefox-i3-container/
